static/js/cohort_filelist.js

```diff
@@ -35,7 +36,8 @@ require.config({
         'jqueryui': ['jquery'],
         'session_security': ['jquery'],
         'assetscore': ['jquery', 'bootstrap', 'jqueryui'],
-        'assetsresponsive': ['jquery', 'bootstrap', 'jqueryui']
+        'assetsresponsive': ['jquery', 'bootstrap', 'jqueryui'],
+        'tokenfield': ['jquery', 'jqueryui']
```

Comment:
I thought you meant token field like on the gene selection page. Having yet more checkbox and file listing duplicated in another spot seems like it might be confusing; the tokenizer seemed like an easier way to represent entries they can remove but not add to (and lets them edit the selection set at the top of the page rather than needing to scroll).

I also just pushed a commit to fix a bug.